Runtime/Scripts/Rooms/Room.cs

```diff
@@ -316,6 +323,18 @@ internal void OnConnect(FfiOwnedHandle roomHandle, RoomInfo info, OwnedParticipa
             FfiClient.Instance.DisconnectReceived += OnDisconnectReceived;
             ConnectionUpdated?.Invoke(this, ConnectionUpdate.Connected);
         }
+        internal void OnLocalTrackPublished(OwnedTrackPublication p)
+        {
+            var publication = trackPublicationFactory.NewTrackPublication(p.Info!);
+
+            var trackHandle = IFfiHandleFactory.Default.NewFfiHandle((IntPtr)p.Handle.Id);
+
+            var track = tracksFactory.NewTrack(trackHandle, TracksFactory.FromTPI2TI(p.Info), this, Participants.LocalParticipant());
+
+            publication.UpdateTrack(track);
+
+            Participants.LocalParticipant().AddTrack(publication);
```

Comment:
Within Room's existing LocalTrackPublished event we already add the publication to LocalParticipant.  It passes the track sid which I think is enough for unsubscribe. 

So what does this do that we weren't already doing?